src/jti_background.js

```diff
@@ -223,6 +231,34 @@ chrome.storage.sync.get(StorageID, function (templates) {
     }
 });
 
+// Listen for when extension is installed or updated
+chrome.runtime.onInstalled.addListener(
+    function (details) {
+        if (details.reason === 'install' || details.reason === 'update') {
```

Comment:
Just curious - how did you test this? Are you able to mock an update or install request?